cioos_data_transform/ios_data_transform/ios_data_transform/tests/test.py

```diff
@@ -1,7 +1,6 @@
 # script runs automated tests on data conversions
 import sys
 import os
-sys.path.insert(0, os.getcwd()+'/../../')
```

Comment:
We need this line in the test.py to use modified code without reinstallation of package.